pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -97,6 +97,7 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 		f.pluginNameToWeightMap[name] = int(pg[name].Weight)
 		if f.pluginNameToWeightMap[name] == 0 {
 			f.pluginNameToWeightMap[name] = 1
+			klog.V(4).Infof("plugin %v is not configured with weight, weight set to 1", name)
```

Comment:
It makes sense to log when this happens, please capitalize the first letter.

Reply:
I checked the other logs in the same class - all of them start with lowercase

Reply:
> I checked the other logs in the same class - all of them start with lowercase

They start with lowercase because the message needs to wrap as errors which should start with lowercase. But this is not a big problem.

/lgtm